### DISCUSSION TITLE
Snapdragon X Elite Support for Windows 11?

### DISCUSSION THREAD

**Discussion by kel656** [phcode-dev/phoenix#1882] [Ideas]
Very good program for coding the web.  Just a thought, do you have any plans to add Windows on ARM support for Snapdragon X Elite processors such as as the new Surface Pro?  When I go to the downloads page I only see Windows x64 yet when for Mac I see both Intel X64 and Apple Silicon varieties.  I can run X64 apps on ARM using Prism  but it however is a bitt on the laggy side with live preview open on ARM CPU machines.

**Comment by abose** [comment]
It should be possible for us to create an ARM release. Not yet prioritized as we are fully on git and terminal integration ATM scheduled to go before December.

@charlypa are we seeing more users for arm64 windows builds?

**Comment by charlypa** [comment]
@abose we don't know how many users are windows on arm. However, this should be doable. we can have a poll to see how may users are interested if we have sufficient numbers we can take it up

**Reply by abose** [reply]
Maybe we can check how many users visit our download page from windows on arm machines. That may give better results of the actual user base relevant to us instead of a poll.
